### PR TITLE
Made PR comments changes

### LINKED ISSUE #13: Fix all PR review comments

### PR DESCRIPTION
**Pull request by amrutsabale**
Fixes #13 

Fixed all PR review comments

1. Moving constants in separate file
2. Use Alert component 
3. Styles fixes
4. Use day.js for dates

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx [hunk 1]
```diff
@@ -33,19 +33,19 @@ export default function ContactTable({
...
+            <th className="text-center">Add to Basecamp</th>
             <th></th>
           </tr>
         </thead>
         <tbody>
           {contacts.map(contact => (
             <tr
               key={contact.id}
-              className={"group cursor-pointer bg-white hover:bg-gray-50"}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx]: Please remove this, `nui-table--hover` adds hover transition effects.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx [hunk 2]
```diff
@@ -34,20 +34,20 @@ export default function NoteTable({
...
+            <th className="text-center">Contact</th>
             <th></th>
           </tr>
         </thead>
         <tbody>
           {notes.map(note => (
             <tr
               key={note.id}
-              className={"group cursor-pointer bg-white hover:bg-gray-50"}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Notes/NoteTable.jsx]: Please remove this `nui-table--hover` adds hover transition effects.

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@amrutsabale Please check the suggestions.

File: app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx (hunk 1 above)
Comment (amrutsabale) [reply app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx]: Removed unwanted css classes.

File: app/javascript/src/components/Dashboard/Notes/NoteTable.jsx (hunk 2 above)
Comment (amrutsabale) [reply app/javascript/src/components/Dashboard/Notes/NoteTable.jsx]: Removed unwanted css classes.

**Pull request merged**